src/lib/concurrency/transaction_manager.cpp

```diff
@@ -40,13 +44,9 @@ void TransactionManager::deregister_transaction(CommitID snapshot_commit_id) {
     return;
   }
 
-  /**
-   * TODO(all) Change Google tests like get_table_test.cpp to always use the TransactionManager for creating
-   * TransactionContexts. After dong that, enable the following check:
-   */
-  //  Fail(opossum::trim_source_file_path(__FILE__) + ":" BOOST_PP_STRINGIZE(__LINE__) +
-  //       " Could not find snapshot_commit_id in TransactionManager's _active_snapshot_commit_ids. Therefore," +
-  //       " the removal failed and the function should not have been called.");
+  Fail(opossum::trim_source_file_path(__FILE__) + ":" BOOST_PP_STRINGIZE(__LINE__) +
```

Comment:
That was not what I meant. Have a look at assert.hpp. There, `Assert` builds the file/line/msg string and passes it to `Fail`. If we move that part to `Fail`, we would have line information for both asserts and fails. Only `assert.hpp` needs to change.

Reply:
In any case, you should probably just do `Assert(it != _active_snapshot_commit_ids.end(), "...")`;

Reply:
> That was not what I meant. Have a look at assert.hpp. There, `Assert` builds the file/line/msg string and passes it to `Fail`. If we move that part to `Fail`, we would have line information for both asserts and fails. Only `assert.hpp` needs to change.

Ok, I did not succeed in changing this right away. I opened a ticket and will take care of it in a week or so. 

For now, I'll change this as @mrzzzrm suggested.